src/mlpack/tests/hmm_test.cpp

```diff
@@ -1827,32 +1831,31 @@ BOOST_AUTO_TEST_CASE(DiagonalGMMHMMLoadSaveTest)
   for (size_t j = 0; j < hmm.Emission().size(); ++j)
   {
     // Check the number of Gaussians.
-    BOOST_REQUIRE_EQUAL(hmm.Emission()[j].Gaussians(),
+    REQUIRE(hmm.Emission()[j].Gaussians() ==
                         hmm2.Emission()[j].Gaussians());
```

Comment:
```suggestion
    REQUIRE(hmm.Emission()[j].Gaussians() == hmm2.Emission()[j].Gaussians());
```
I think this can fit on one line now. :+1: